ui/src/shared/components/inlineLabels/InlineLabels.tsx

```diff
@@ -101,3 +111,12 @@ export default class InlineLabels extends Component<Props> {
     onRemoveLabel(label)
   }
 }
+
+const mstp = (state: AppState, props: OwnProps) => {
+  const labels = getAll<Label>(state, ResourceType.Labels)
```

Comment:
This is just being passed to a conditionally rendered InlineLabelsEditor, which is itself connected to redux. I think it would be more efficient for InlineLabelsEditor to get a list of all labels itself